crates/ruff_linter/src/rules/flake8_type_checking/mod.rs

```diff
@@ -35,6 +35,8 @@ mod tests {
     #[test_case(Rule::RuntimeImportInTypeCheckingBlock, Path::new("TCH004_8.py"))]
     #[test_case(Rule::RuntimeImportInTypeCheckingBlock, Path::new("TCH004_9.py"))]
     #[test_case(Rule::RuntimeImportInTypeCheckingBlock, Path::new("quote.py"))]
+    #[test_case(Rule::UnquotedTypeAlias, Path::new("TCH007.py"))]
+    #[test_case(Rule::QuotedTypeAlias, Path::new("TCH008.py"))]
```

Comment:
If I'm not mistaken, in order to accomplish this all I would have to do is to generate linter settings that enable both rules, since the test utility functions already make sure that fixes converge after a few iterations, correct?

Reply:
Yes, I think you're right!